crypto/provider_core.c

```diff
@@ -964,6 +964,12 @@ static int provider_init(OSSL_PROVIDER *prov)
                        prov->name);
         goto end;
     }
+#ifndef FIPS_MODULE
```

Comment:
Why are trace statements above/before not "FIPS-#ifdef guarded"?

Reply:
There was one that was missed, I'm fixing it